cliutils/distribution/distribute.go

```diff
@@ -0,0 +1,52 @@
+package distribution
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/common/spec"
+	pluginsCommon "github.com/jfrog/jfrog-cli-core/v2/plugins/common"
+	"github.com/jfrog/jfrog-cli-core/v2/plugins/components"
+	distributionUtils "github.com/jfrog/jfrog-client-go/utils/distribution"
+)
+
+func CreateDefaultDistributionRules(c *components.Context) *spec.DistributionRules {
```

Comment:
This file already exists in distribution/cli/distribute.go

Reply:
Moved it from cli-core since it is related to lifecycle and distribution commands. I couldnt find it in `cli-core/common/cliutils` is it part of any other package?

Reply:
It exists in the [CLI](https://github.com/jfrog/jfrog-cli/blob/dev/utils/distribution/distribute.go). Please delete the duplication and use the one in this PR.

Reply:
yeah but the link above points to a different branch, the [PR](https://github.com/jfrog/jfrog-cli/pull/2878/files#diff-5c1add75e996fb79bdaadcf72c7942f2e811d4a634b2956be913bba3583d6076) I have raised from `migrate-life-cycle-commands` on jfrog-cli already takes care of it. Are you referring to any other file?